packages/node_modules/@ciscospark/redux-module-media/src/reducer.js

```diff
@@ -23,8 +23,8 @@ export const initialState = fromJS({
     call: null
   },
   callState: {
-    sendingAudio: true,
-    sendingVideo: true
+    sendingAudio: false,
+    sendingVideo: false
```

Comment:
Did you test this with a slow connection to make sure thing still look right? We need to make sure that there's no chance that the buttons show up in the `false` state.

There was also a case where buttons didn't show up at all if there is any sort of connection issue during the dialing phase and the user gets stuck without a way to hangup the call.